dist/index.html

```diff
@@ -51,7 +51,7 @@ <h2 class="moto">CUSTOM DESIGNS · MARKETING · BRANDING · AND MORE</h2>
               CONTACT
             </div>
             <!-- TODO: Replace with static horizontal rule asset when delivered by designer. -->
-            <hr class="fancy">
+            <img class="fancy" src="/Users/landonmetcalf/Documents/coding/levelUpLabs/preImmersive/preCourseWork/seek-a-design-playground/dist/assets/other/OrnamentalDivider.svg" />
```

Comment:
You've got a bug here! You're referring to the SVG's file path with an **absolute path** instead of a **relative path**.

This means that the path will only work on your local machine! When I run the app, I can't see the `/Users/landonmetcalf/...` directory.

Switch this to:

```
`src="./assets/other/OrnamentalDivider.svg"` 
```

and we'll be good to go!

Reply:
**Uh oh!**

You also forgot to commit the `OrnamentalDivider.svg` file that you added to your repo. Can you please add the file  to the path referenced in the comment above?